deployments/roman/image/environments/cvt/cvt-frozen.yml

```diff
@@ -0,0 +1,303 @@
+  - pyparsing=2.4.7
+  - pyrsistent=0.17.3
+  - pysocks=1.7.1
+  - pytables=3.6.1
+  - python=3.8.6
+  - python-dateutil=2.8.1
+  - python-editor=1.0.4
+  - python-json-logger=2.0.1
+  - python-libarchive-c=2.9
+  - python_abi=3.8
+  - pytz=2020.4
+  - pywavelets=1.1.1
+  - pyyaml=5.3.1
+  - pyzmq=20.0.0
+  - readline=8.0
+  - requests=2.25.0
+  - ripgrep=12.1.1
+  - ruamel.yaml=0.16.12
+  - ruamel.yaml.clib=0.2.2
+  - ruamel_yaml=0.15.80
+  - scikit-image=0.17.2
+  - scikit-learn=0.23.2
+  - scipy=1.5.3
+  - seaborn=0.11.0
+  - seaborn-base=0.11.0
+  - send2trash=1.5.0
+  - setuptools=49.6.0
+  - six=1.15.0
+  - snappy=1.1.8
+  - sortedcontainers=2.3.0
+  - soupsieve=2.0.1
+  - sqlalchemy=1.3.20
+  - sqlite=3.33.0
+  - statsmodels=0.12.1
+  - sympy=1.6.2
+  - tblib=1.6.0
+  - terminado=0.9.1
+  - testpath=0.4.4
+  - threadpoolctl=2.1.0
+  - tifffile=2020.10.1
+  - tini=0.18.0
+  - tk=8.6.10
+  - toolz=0.11.1
+  - tornado=6.1
+  - tqdm=4.52.0
+  - traitlets=5.0.5
+  - typing_extensions=3.7.4.3
+  - urllib3=1.25.11
+  - vincent=0.4.4
+  - wcwidth=0.2.5
+  - webencodings=0.5.1
+  - websockify=0.9.0
+  - wheel=0.35.1
+  - widgetsnbextension=3.5.1
+  - xlrd=1.2.0
+  - xz=5.2.5
+  - yaml=0.2.5
+  - zeromq=4.3.3
+  - zfp=0.5.5
+  - zict=2.0.0
+  - zipp=3.4.0
+  - zlib=1.2.11
+  - zstd=1.4.5
+  - pip:
+    - aiohttp==3.7.3
+    - alabaster==0.7.12
+    - ansiwrap==0.8.4
+    - appdirs==1.4.4
+    - astropy-sphinx-theme==1.1
+    - async-timeout==3.0.1
+    - awscli==1.18.181
+    - babel==2.9.0
+    - black==20.8b1
+    - boto3==1.16.21
+    - botocore==1.19.21
+    - ci-watson==0.5
+    - codecov==2.1.10
+    - colorama==0.4.3
+    - coverage==5.3
+    - docutils==0.15.2
+    - flake8==3.8.4
+    - imagesize==1.2.0
+    - iniconfig==1.1.1
+    - jmespath==0.10.0
+    - jupyter-desktop-server==0.1.2
+    - jupyter-server-proxy==1.5.0
+    - mccabe==0.6.1
+    - multidict==5.0.2
+    - mypy-extensions==0.4.3
+    - nbgitpuller==0.9.0
+    - numpydoc==1.1.0
+    - papermill==2.2.2
+    - pathspec==0.8.1
+    - pluggy==0.13.1
+    - py==1.9.0
+    - pyasn1==0.4.8
+    - pycodestyle==2.6.0
+    - pyflakes==2.2.0
+    - pytest==6.1.2
+    - pytest-cov==2.10.1
+    - pytest-doctestplus==0.8.0
+    - pytest-openfiles==0.5.0
+    - regex==2020.11.13
+    - requests-mock==1.8.0
+    - rsa==4.5
+    - s3transfer==0.3.3
+    - simpervisor==0.3
+    - snowballstemmer==2.0.0
+    - sphinx==3.3.1
+    - sphinx-asdf==0.1.0rc8
+    - sphinx-astropy==1.3
+    - sphinx-automodapi==0.13
+    - sphinx-bootstrap-theme==0.7.1
+    - sphinx-gallery==0.8.1
+    - sphinx-rtd-theme==0.5.0
+    - sphinxcontrib-applehelp==1.0.2
+    - sphinxcontrib-devhelp==1.0.2
+    - sphinxcontrib-htmlhelp==1.0.3
+    - sphinxcontrib-jsmath==1.0.1
+    - sphinxcontrib-qthelp==1.0.3
+    - sphinxcontrib-serializinghtml==1.1.4
+    - stsci-rtd-theme==0.0.2
+    - tenacity==6.2.0
+    - textwrap3==0.9.2
+    - toml==0.10.2
+    - typed-ast==1.4.1
+    - yarl==1.6.3
+prefix: /opt/conda/envs/cvt
```

Comment:
What does "prefix" do?

Reply:
prefix is defining where that environment  lives in conda